### PR TITLE
Add Error handling on duplicate dashboard names 

### PR DESCRIPTION
**Pull request by rupal-bq**
### Description
- added check if dashboard with same name already exists

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by rupal-bq** [comment]
depends on https://github.com/opensearch-project/dashboards-observability/pull/415.

**Comment by ps48** [comment]
@rupal-bq Can you please re-base your PR to #435. We added few things on top of #415, in that PR.

File: .cypress/integration/3_panels.spec.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ describe('Creating visualizations', () => {
   });
 });
 
-describe('Testing panels table', () => {
+describe.only('Testing panels table', () => {
```
Comment (derek-ho) [inline .cypress/integration/3_panels.spec.ts:80]: can remove/revert this file to without .only console and commented out code

**Comment by pjfitzgibbons** [comment]
@rupal-bq #435 (superceded #415) is now merged to 2.7.  So just reabase this PR from 2.7 and you'll be ready.

**Comment by rupal-bq** [comment]
created https://github.com/opensearch-project/dashboards-observability/pull/441

**Pull request closed without merging**